CMakeLists.txt

```diff
@@ -31,6 +31,11 @@ endif (CMAKE_VERSION VERSION_LESS 3.9)
 set (CMAKE_MACOSX_RPATH 1)
 set (CMAKE_INSTALL_RPATH "${CMAKE_INSTALL_PREFIX}/lib")
 
+set (
+	CMAKE_INSTALL_DEFAULT_COMPONENT_NAME
+	"elektra-misc"
+	CACHE STRING "Default component name" FORCE)
```

Comment:
Does this really need to be in the cache? If yes, at least use mark_as_advanced as it is not something a normal user wants to change.

Reply:
Apparently a previous configuration issue led me to cache this variable. This is definetly not needed and a mistake from my side. I will fix this in the upcoming PR, since this one is already merged, if this is fine for you.

Reply:
Yes, of course it can be fixed with the next PR. But it would be good if it will be fixed before the release as cmake cache variables are user-facing (to maintainers) and should be documented and so on. So better to never introduce this cache variable to any release. Anyway: would be great to have RPM+DEB for next release. Is it your plan to do so?

Reply:
I was currently working on extending the release automation (ABI tests, external test suites etc) but I can of course also create RPM packages for this release. With our current CPack setup this probably shouldn't be that much work.

Reply:
#3519